### PR TITLE
Add the GHS.JEM

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

Add the GHS.JEM

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ghs/jem/config.h [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/ghs/jem/config.h:84]:
```suggestion
/*== choose animations ==*/
#    define RGBLIGHT_EFFECT_BREATHING
#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#    define RGBLIGHT_EFFECT_SNAKE
#    define RGBLIGHT_EFFECT_KNIGHT
#    define RGBLIGHT_EFFECT_CHRISTMAS
#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
#    define RGBLIGHT_EFFECT_RGB_TEST
#    define RGBLIGHT_EFFECT_ALTERNATING
```

File: keyboards/ghs/jem/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/ghs/jem/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/ghs/jem/rules.mk [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/ghs/jem/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
```

File: keyboards/ghs/jem/rules.mk [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/ghs/jem/rules.mk:18]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ramonimbao** [comment]
Hello, can I request for a recheck of this? I've finished migrating everything to the new data-driven way of QMK.

File: keyboards/ghs/jem/soldered/config.h [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Ramon Imbao (@ramonimbao)
```
Comment (fauxpark) [inline keyboards/ghs/jem/soldered/config.h:1]: This file can be deleted.

File: keyboards/ghs/jem/hotswap_ansi/keymaps/default/readme.md [hunk 6]
```diff
@@ -0,0 +1 @@
+# The default keymap for GHS.JEM
```
Comment (fauxpark) [inline keyboards/ghs/jem/hotswap_ansi/keymaps/default/readme.md:1]: This file can be deleted.

File: keyboards/ghs/jem/hotswap_ansi/config.h [hunk 7]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Ramon Imbao (@ramonimbao)
```
Comment (fauxpark) [inline keyboards/ghs/jem/hotswap_ansi/config.h:1]: This file can be deleted.

File: keyboards/ghs/jem/soldered/keymaps/default/readme.md [hunk 8]
```diff
@@ -0,0 +1 @@
+# The default keymap for GHS.JEM
```
Comment (fauxpark) [inline keyboards/ghs/jem/soldered/keymaps/default/readme.md:1]: This file can be deleted.

File: keyboards/ghs/jem/info.json [hunk 9]
```diff
@@ -0,0 +1,272 @@
+{
+    "keyboard_name": "GHS.JEM",
+    "manufacturer": "Gone Hacking Studio",
+    "url": "",
+    "maintainer": "ramonimbao",
```
Comment (fauxpark) [inline keyboards/ghs/jem/info.json:5]:
```suggestion
    "maintainer": "ramonimbao",
    "usb": {
        "vid": "0x0645",
        "device_version": "1.0.0"
    },
```

File: keyboards/ghs/jem/hotswap_ansi/info.json [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{
+    "usb": {
+        "vid": "0x0645",
+        "pid": "0x2003",
+        "device_version": "1.0.0"
```
Comment (fauxpark) [inline keyboards/ghs/jem/hotswap_ansi/info.json:5]:
```suggestion
        "pid": "0x2003"
```

File: keyboards/ghs/jem/soldered/info.json [hunk 11]
```diff
@@ -0,0 +1,7 @@
+{
+    "usb": {
+        "vid": "0x0645",
+        "pid": "0x1003",
+        "device_version": "1.0.0"
```
Comment (fauxpark) [inline keyboards/ghs/jem/soldered/info.json:5]:
```suggestion
        "pid": "0x1003"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**